hack/release.toml

```diff
@@ -6,7 +6,7 @@ github_repo = "siderolabs/talos"
 match_deps = "^github.com/((talos-systems|siderolabs)/[a-zA-Z0-9-]+)$"
 
 # previous release
-previous = "v1.0.3"
+previous = "v1.0.4"
```

Comment:
@smira just wanted to make sure i got this file right

Reply:
Yep, this looks good!